content/en/agent/logs/advanced_log_collection.md

```diff
@@ -442,6 +443,29 @@ The example above will match `/var/log/myapp/log/myfile.log` but `/var/log/myapp
 
 **Note**: The Agent requires read and execute permissions on a directory to list all the available files in it.
 
+## Encode UTF-16 format logs
+
+If applications logs are written in UTF-16 format, from Datadog Agent **v6.23/v7.23** user can encode these logs so that they are parsed as expected in [Logs Explorer](https://docs.datadoghq.com/logs/explorer/#overview). Use the `encoding` parameter in logs configuration section. Set it to `utf-16-le` for UTF16 little-endian and `utf-16-be` for UTF16 big-endian. Any other value will be ignored and the Agent will read the file as UTF8.
+
```

Comment:
Nit pick, the exemple is not specially related to the UTF-16 section. Could we put in the head of the section instead?

Reply:
Hi @pcarioufr , can you please elaborate a bit more on where do you want me to insert the example configuration. What do you mean by the 'head of the section'?